gemrb/core/Audio/Backend.h

```diff
@@ -0,0 +1,103 @@
+/* GemRB - Infinity Engine Emulator
+ * Copyright (C) 2025 The GemRB Project
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#ifndef H_AUDIO_BACKEND
+#define H_AUDIO_BACKEND
```

Comment:
I propose the filenames be renamed to AudioBackend.* as well, not just to match the class and guard, but to be less generic.

Reply:
Same for Settings.h.